active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,336 @@
+
+XAML
+```XAML
+<Grid>
+        <!-- DataGrid Layout -->
+        <controls:PagerControl x:Name="MainPagerControl"
+                DisplayMode="Auto"
+                NumberOfPages="10"
+                FirstButtonVisibility="None"
+                PreviousButtonVisibility="AlwaysVisible"
+                NextButtonVisibility="AlwaysVisible"
+                LastButtonVisibility="None"
+                Prefix="Page"
+                Suffix="of 10"/>
+        </controls:PagerControl>
+</Grid>
+```
+
+Below is the XAML for how to add the numerical button panel display mode. This mode is not part of the auto mode and needs to be explicitly set. The prefix and suffix properties do not apply to this mode so they will be ignored if they are set. The ButtonPanelShowFirstAndLast property is specific to this mode and controls if the button panel will always show the first and last page. This will also display ellipsis before the last page and after the first page to indicate that there are pages between the page numbers being shown and the last page. They will automatically be hidden if the pages being shown are close to the beggining or end of the page range. 
+
+XAML
+```XAML
+<Grid>
+        <!-- ListView Layout -->
+        <controls:PagerControl x:Name="MainPagerControl"
+                DisplayMode="NumericalButtonPanel"
+	        NumberOfPages="10"
+                FirstButtonVisibility="None"
+                PreviousButtonVisibility="AlwaysVisible"
+                NextButtonVisibility="AlwaysVisible"
+                LastButtonVisibility="None"
+	        EllipsisShowFirstAndLast="True">
+	</controls:PagerControl>
+</Grid>
+```
+
+### Customize edge buttons
+
+You can customize the visibility of each of the edge buttons (the first, previous, next, and last) by setting their visibility properties. In this example, the first and previous buttons will always be visible while the next button will be hidden when the last page is selected. The last button will be hidden. By default, all buttons will be set to AlwaysVisible.
+
+XAML
+```XAML
+<Grid>
+        <!-- ListView Layout -->
+        <controls:PagerControl x:Name="MainPagerControl"
+                DisplayMode="ComboBox"
+                NumberOfPages="10"
+                FirstButtonVisibility="AlwaysVisible"
+                PreviousButtonVisibility="AlwaysVisible"
+                NextButtonVisibility="HiddenOnEdge"
+                LastButtonVisibility="None"
+                Prefix="Page"
+                Suffix="of 10"/>
+        </controls:PagerControl>
+</Grid>
+```
+
+### Handling a page change event
+
+When a user interacts with the pager control and selects an edge button or page number, the page changed event will be fired. This event will provide the index the user is coming from (previousPage) and the index the user is navigating to (currentPage). Below is a code example of how to handle this event. 
```

Comment:
What if the user presses next/previous button? Or is this part of "edge buttons"?

Reply:
I think it would count as edge button here, but that's not really the best name, I am not sure if we need to call out specific ways to interact here. There aren't many ways to interact with the PagerControl and all of them should change the selected index.